tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -434,6 +436,21 @@ def test_polygons(self):
             self.assertEqual(poly.wkt, Polygon(*tuple(r for r in poly)).wkt)
             self.assertEqual(poly.wkt, Polygon(*tuple(LinearRing(r.tuple) for r in poly)).wkt)
 
+    def test_polygons_templates(self):
```

Comment:
An attempt at a regression test for https://code.djangoproject.com/ticket/4740 but not sure it captures the issue.